apps/laboratory/src/pages/library/wagmi-siwe.tsx

```diff
@@ -19,6 +19,7 @@ const modal = createWeb3Modal({
   enableAnalytics: true,
   metadata: ConstantsUtil.Metadata,
   siweConfig,
+  enableOnramp: true,
```

Comment:
Is this onramp flag related with this PR? This should be true by default that we recently did this. Could you double check?

Reply:
Not related not sure how it ended up here. I'll double check 👍